src/mlj_model_interface.jl

```diff
@@ -59,6 +59,7 @@ function MLJModelInterface.fit(model::MLJFluxModel,
         build(model, rng, shape) |> move
     catch ex
         @error ERR_BUILDER
+        throw(ex)
```

Comment:
```suggestion
        rethrow()
```
Lets use rethrow here as its more cleaner